src/20240917_AaveV3Ethereum_MayFundingUpdatePartB/AaveV3Ethereum_MayFundingUpdatePartB_20240917.sol

```diff
@@ -0,0 +1,94 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import {IProposalGenericExecutor} from 'aave-helpers/src/interfaces/IProposalGenericExecutor.sol';
+import {IERC20} from 'solidity-utils/contracts/oz-common/interfaces/IERC20.sol';
+import {SafeERC20} from 'solidity-utils/contracts/oz-common/SafeERC20.sol';
+import {AaveV2Ethereum, AaveV2EthereumAssets} from 'aave-address-book/AaveV2Ethereum.sol';
+import {AaveV3Ethereum, AaveV3EthereumAssets} from 'aave-address-book/AaveV3Ethereum.sol';
+import {CollectorUtils} from 'aave-helpers/src/CollectorUtils.sol';
+
+/**
+ * @title May Funding Update Part B
+ * @author karpatkey_TokenLogic
+ * - Snapshot: Direct-to-AIP
+ * - Discussion: https://governance.aave.com/t/arfc-may-funding-update/17768/5
+ */
+contract AaveV3Ethereum_MayFundingUpdatePartB_20240917 is IProposalGenericExecutor {
+  using SafeERC20 for IERC20;
+
+  address public constant COLLECTOR = address(AaveV3Ethereum.COLLECTOR);
+  address public constant POOL_V2 = address(AaveV2Ethereum.POOL);
+  address public constant POOL_V3 = address(AaveV3Ethereum.POOL);
+  address public constant RECIPIENT_500 = 0x1770776deB0A5CA58439759FAdb5cAA014501241;
+  address public constant RECIPIENT_1000 = 0x7dF98A6e1895fd247aD4e75B8EDa59889fa7310b;
+  address public constant IMMUNEFY = 0x7119f398b6C06095c6E8964C1f58e7C1BAa79E18;
+  uint256 public constant RECIPIENT_500_AMOUNT = 500e18;
```

Comment:
It doesn't really make sense to:
1. Define a constant including the amount in the naming.
2. Assign the constant value.
3. Use the constant below.

Given that you use set the amount on the object of the migrations, better to also hardcode the amount directly on the transfer below. Probably even the recipient too